test/src/main/java/org/axonframework/test/aggregate/ResultValidatorImpl.java

```diff
@@ -115,6 +115,36 @@ public ResultValidator expectReturnValueMatching(Matcher<?> matcher) {
         return this;
     }
 
+    @Override
+    public ResultValidator expectExceptionMessage(Matcher<?> exceptionMessageMatcher) {
+        StringDescription description = new StringDescription(
+                new StringBuilder("Given exception message matcher is null!"));
+        if (exceptionMessageMatcher == null) {
+            reporter.reportWrongExceptionMessage(actualException, description);
+            return this;
+        }
+        exceptionMessageMatcher.describeTo(description);
+        if (actualException != null && !exceptionMessageMatcher.matches(actualException.getMessage())) {
+            reporter.reportWrongExceptionMessage(actualException, description);
+        }
+        return this;
+    }
+
+    @Override
+    public ResultValidator expectExceptionMessage(String exceptionMessage) {
```

Comment:
Curious: doesn't it make sense to call the `ResultValidator expectExceptionMessage(Matcher<?>)` function from this `ResultValidator expectExceptionMessage(String)` function?

The only real difference is this function does an equals and the other calls `matches()`.

Reply:
Great remark!

Reply:
Nice adjustment as well :)